analyzers/its/Projects/ManuallyAddedNoncompliantIssues.CS/AspNet4x/Web.Release.config

```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0"?>
```

Comment:
Maybe we can also keep only ony Web.config file?

Reply:
Yes, the app still compiles and starts. I have removed both `Web.Release.config` and `Web.Debug.config`.